Tests/PklSwiftTests/ProjectTest.swift

```diff
@@ -39,6 +40,20 @@ class ProjectTest: XCTestCase {
 
         let file = URL(fileURLWithPath: NSTemporaryDirectory(), isDirectory: true)
             .appendingPathComponent("PklProject")
+        let httpSetting = version < pklVersion0_26 ? "" : """
+            http {
+              proxy {
+                address = "http://proxy.mock.example.com:1"
```

Comment:
Same comment; for here and line 56: can we use `localhost` as the proxy address? `example.com` is a real domain.